gitbootcamp.rst

```diff
@@ -24,24 +25,61 @@ You'll only need to do this once.
 
 4. Your fork will be created at https://github.com/<username>/cpython.
 
+.. _clone-your-fork:
 
 Cloning The Forked CPython Repository
 -------------------------------------
 
 You'll only need to do this once.  From your command line::
 
    $ git clone git@github.com:<username>/cpython.git
+
+It is also recommended to configure an ``upstream`` remote::
+
    $ cd cpython
    $ git remote add upstream git@github.com:python/cpython.git
 
+You can also use SSH-based or HTTPS-based URLs.
 
 Listing the Remote Repositories
 -------------------------------
 
-To list the remote repositories that are configured, along with their urls::
+To list the remote repositories that are configured, along with their URLs::
 
    $ git remote -v
 
+You should have two remotes: ``origin`` pointing to your fork,
+and ``upstream`` pointing to the official CPython repository::
+
+   origin  git@github.com:<your-username>/devguide.git (fetch)
+   origin  git@github.com:<your-username>/devguide.git (push)
+   upstream        git@github.com:python/devguide.git (fetch)
```

Comment:
Since this doc will often be used by contributors that do not have push privileges for CPython, the upstream remote should point to `https://github.com/python/cpython` in the doc here.

Perhaps a note for committers that they should use what you have here for upstream.

Reply:
AFAICT, with this URL, they still should be able to fetch.  If they try to push they will get an error because they don't have privileges, but that's OK since they are not supposed to push on upstream anyway.  Unless I'm missing something, this URL should be fine for both contributors and core devs.

Reply:
Thanks @willingc  and @ezio-melotti. I'm thinking it's fine to leave the `upstream` using SSH url.
For contributors, they would have to configure SSH key anyway.